docs/product-user-guide/src/main/asciidoc/interacting-with-processes-starting-stopping-tasks-proc.adoc

```diff
@@ -13,15 +13,20 @@ Ensure that you are logged in and have appropriate permissions for starting and
 
```

Comment:
In the line 4 above, I guess for consistency reasons with the steps in the procedure, it should be written **Task Inbox** page as well.